packages/service-client/odsp-client/README.md

```diff
@@ -0,0 +1,101 @@
+
+const connectionConfig: OdspConnectionConfig = {
+	tokenProvider: "<YOUR_TOKEN_PROVIDER>",
+	siteUrl: "<SITE_URL>",
+	driveId: "<RAAS_DRIVE_ID>",
+	path: "<FOLDER_PATH>",
+};
+
+export const clientProps: OdspClientProps = {
+	connection: connectionConfig,
+};
+
+const client = new OdspClient(clientProps);
+```
+
+### Experimental Features
+
+`OdspClient` supports the ability to instantiate with experimental features enabled.
+These features are experimental in nature and should **NOT** be used in production applications.
+To learn more, see [Experimental Features](https://fluidframework.com/docs/build/experimental-features/).
+
+## Fluid Containers
+
+A Container instance is a organizational unit within Fluid. Each Container instance has a connection to the defined Fluid Service and contains a collection of collaborative objects.
+
+Containers are created and identified by unique IDs. Management and storage of these IDs are the responsibility of the developer.
+
+## Defining Fluid Containers
+
+Fluid Containers are defined by a schema. The schema includes initial properties of the Container as well as what collaborative objects can be dynamically created.
+
+```typescript
+const containerSchema = {
+	initialObjects: {
+		/* ... */
+	},
+	dynamicObjectTypes: [
+		/*...*/
+	],
+};
+const OdspClient = new OdspClient(clientProps);
+const { container, services } = await OdspClient.createContainer(containerSchema);
+
+const itemId = await container.attach();
+const attributes = await services.tenantAttributes();
+const sharingUrl = attributes.sharingUrl;
+const driveId = attributes.driveId;
+```
+
+## Using Fluid Containers
+
+Using the `OdspClient` object the developer can create and get Fluid containers. Because Fluid needs to be connected to a server, containers need to be created and retrieved asynchronously.
+
+```typescript
+import { OdspClient } from "@fluid-experimental/odsp-client";
+
+const OdspClient = new OdspClient(props);
+const { container, services } = await OdspClient.getContainer("_unique-id_", schema);
+```
+
+## Using initial objects
+
+The most common way to use Fluid is through initial collaborative objects that are created when the Container is created.DistributedDataStructures and DataObjects are both supported types of collaborative objects.
+
+`initialObjects` are loaded into memory when the Container is loaded and the developer can access them via the Container's `initialObjects` property. The `initialObjects` property has the same signature as the Container schema.
+
+```typescript
+// Define the keys and types of the initial list of collaborative objects.
+// Here, we are using a SharedMap DDS on key "map1" and a SharedString on key "text1".
+const schema = {
+	initialObjects: {
+		map1: SharedMap,
+		text1: SharedString,
+	},
+};
+
+// Fetch back the container that had been created earlier with the same url and schema
+const { container, services } = await OdspClient.getContainer("_unique-url_", schema);
```

Comment:
It will be hard for developers to use this flow, without schema being stored as part of the file. How would developer go by changing that schema, and supporting V1, V2, ... files? Any project have to go through it eventually, it's not the question of IF, it's a question of WHEN